documentation/book/ref-kafka-bridge-tls.adoc

```diff
@@ -20,10 +20,10 @@ spec:
   # ...
   tls:
     trustedCertificates:
-      - secretName: my-secret
-        certificate: ca.crt
-      - secretName: my-other-secret
-        certificate: certificate.crt
+    - secretName: my-secret
```

Comment:
From `examples/kafka-connect/kafka-connect.yaml`
```tls:
    trustedCertificates:
      - secretName: my-cluster-cluster-ca-cert
        certificate: ca.crt
```
Are we sure about the indentation?

Reply:
Both are valid and working. This:

```yaml
  tls:
    trustedCertificates:
      - secretName: my-cluster-cluster-ca-cert
        certificate: ca.crt
```

has the same meaning as this:

```yaml
  tls:
    trustedCertificates:
    - secretName: my-cluster-cluster-ca-cert
      certificate: ca.crt
```

I usually prefer the first one as I find it more readable. But I think that most editors will format into the second. Either way in both case the secret reference will be in the hierarchy under the `trustedCertificates`.